synapse/models/files.py

```diff
@@ -556,6 +577,54 @@ def getModelDefs(self):
                     ('base:ext', ('str', {}), {'ro': True,
                         'doc': 'The file extension.'}),
                 )),
+
+                ('file:mime:macho:loadcmd', {}, (
+                    ('file', ('file:bytes', {}), {
+                        'doc': 'The Mach-O file containing the load command.'}),
+                    ('type', ('int', {'enums': s_l_macho.getLoadCmdTypes()}), {
+                        'doc': 'The type of the load command'}),
+                    ('size', ('int', {}), {
+                        'doc': 'The size of the load command structure in bytes.'}),
+                )),
+                ('file:mime:macho:version', {}, (
+                    ('loadcmd', ('file:mime:macho:loadcmd', {}), {
+                        'doc': 'The associated load command node.'}),
+                    ('version', ('str', {}), {
+                        'doc': 'The version of the Mach-O file encoded in an LC_VERSION load command'})
+                )),
+                ('file:mime:macho:uuid', {}, (
+                    ('loadcmd', ('file:mime:macho:loadcmd', {}), {
+                        'doc': 'The actual load command '}),
+                    ('uuid', ('str', {}), {
+                        'doc': 'The UUID of the Mach-O application (as defined in an LC_UUID load command)'}),
+                )),
+                ('file:mime:macho:segment', {}, (
+                    ('loadcmd', ('file:mime:macho:loadcmd', {}), {
+                        'doc': 'The actual load command node.'}),
+                    ('loadcmd:file', ('file:bytes', {}), {
+                        'doc': 'The Mach-O file the segment is located in'}),
```

Comment:
Did we add this in as an optimization of some kind?

Reply:
Pretty much. It feel out from the discussion iirc. It exists to save a pivot and that's about it.